config/serverless.yml

```diff
@@ -42,7 +42,7 @@ management.deeplinks.navLinkStatus: visible
 
 # Other disabled plugins
 #xpack.canvas.enabled: false #only disabable in dev-mode
-xpack.reporting.enabled: false
+xpack.reporting.enabled: true 
```

Comment:
Let's remove this line, because `true` is the default, and it looks like this section of the file should be for disabled plugins.